src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -1250,7 +1250,7 @@ internal static string GetWindowsPowerShellModulePath()
             }
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-windows`
TestCustomPipeNameCreation
```powershell
Assert.True() Failure
Expected: True
Actual:   False
   at PSTests.Parallel.NamedPipeTests.TestCustomPipeNameCreation() in D:\a\1\s\test\xUnit\csharp\test_NamedPipe.cs:line 25
```